geode-core/src/integrationTest/java/org/apache/geode/internal/AvailablePortHelperIntegrationTest.java

```diff
@@ -278,6 +284,10 @@ public void initializeUniquePortRange_willReturnSamePortsForSameRange(
   @Parameters({"true", "false"})
   public void initializeUniquePortRange_willReturnUniquePortsForUniqueRanges(
       final boolean useMembershipPortRange) {
+    assumeTrue(
+        "Windows has ports scattered throughout the range that makes this test difficult to pass consistently",
+        SystemUtils.isWindows());
```

Comment:
seems like you intended to skip this test on windows, shouldn't it be `assumeFalse` instead?